packages/runtime/src/runtimes/react/components/PreviewProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactNode, useEffect, useMemo } from 'react'
+
+import { StoreContext, storeContextDefaultValue } from '..'
+import * as ReactBuilderPreview from '../../../state/react-builder-preview'
+import * as ReactPage from '../../../state/react-page'
+import { MakeswiftProvider, MakeswiftClient } from '../../../api/react'
+import { registerDocumentEffect } from '../../../state/actions'
+
+type Props = {
+  client: MakeswiftClient
+  rootElements?: Map<string, ReactPage.Element>
```

Comment:
Is `Element` not available via `ReactBuilderPreview`?

Reply:
Not available, I think it's fine to load `ReactPage` from `PreviewProvider`, as that's how it behaved previously.